web-client/public/js/upload.js

```diff
@@ -367,7 +151,7 @@ export const upload = function (sliderObject, sliderGroupController, drawGraph,
         var validated = edgeWeightNormalizationInputValidation(value);
         $("#normalization-max").val(validated);
         $("#edge-weight-normalization-factor-menu").val(validated);
-        drawGraph(currentNetwork, sliders, nodeColoring);
+        drawGraph(uploadState.currentNetwork, sliders, nodeColoring);
```

Comment:
OK, _mostly_ deletions. The big transition here is that `currentNetwork` is no more; it has been fully transferred to `grnState`. `uploadState` is a temporary holding place for `currentNetwork`, to eventually be replaced fully by `grnState` as the transition progresses.